plugin/PlayAssetDelivery/src/test/java/com/google/play/core/godot/assetpacks/PlayAssetDeliveryTestHelper.java

```diff
@@ -46,4 +53,45 @@ public static Dictionary createAssetPackLocationsDictionary() {
     testDict.put("location2", innerDict2);
     return testDict;
   }
+
+  /**
+   * Mock object factory that returns a mock Task<T> object. Will invoke onSuccessListener with
+   * result if addOnSuccessListener() is called.
+   *
+   * @param result object to be passed to the onSuccessListener
+   * @param <T> parameterized type of the Task object
```

Comment:
nit: `<T> parameterized type of the result returned by this task if it succeeds.`